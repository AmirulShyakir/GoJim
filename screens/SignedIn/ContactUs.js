import React from "react";
import {
  Image,
  StyleSheet,
  Text
} from "react-native";

import MainContainer from "../../components/containers/MainContainer";
import { colours } from "../../components/ColourPalette";
const { primary, secondary, tertiary, white, black, action } = colours;

const ContactUs = ({navigation}) => {
  return (
    <MainContainer>
      <Text style={styles.text}>Hello there</Text>
      <Text style={styles.text2} testID={"text"}>We are Team Sikkunt. We have developed GoJim as an integrated 
      booking app for all NUS facilities for our Orbital project. {"\n\n"}Thanks for downloading 
      GoJim. We appreciate any feedback and you may send them over at our emails. {"\n\n"}-Team Sikkunt
      {"\n\n"}Amirul Shyakir  E0725279@u.nus.edu{"\n"}Muhammad Mursyid  E0725270@u.nus.edu
      {"\n\n"}
      </Text>
      <Image
          testID="image"
          source={require("../../assets/GOJIM.png")}
          style={{
            width: 80,
            height: 50,
            resizeMode: "contain",
            alignSelf: "center",
            position: "absolute",
            bottom: 20,
          }}
        />
    </MainContainer>
  );
};


export default ContactUs;

const styles = StyleSheet.create({
  text: {
    color: white,
    textAlign: "center",
    fontSize: 25,
    fontWeight: "bold",
    marginBottom: 20,
    
  },
  text2: {
    color: white,
    fontSize: 15,
  },
});