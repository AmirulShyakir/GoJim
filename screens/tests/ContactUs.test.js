import React from "react";
import renderer from 'react-test-renderer';
import ContactUs from "../SignedIn/ContactUs";

test("ContactUs Snapshot test", () => {
    const tree = renderer.create(<ContactUs />).toJSON();
  expect(tree).toMatchSnapshot();
});
