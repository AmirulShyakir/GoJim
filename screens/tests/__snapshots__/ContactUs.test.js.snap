// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs Snapshot test 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#1c1515",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 15,
      },
    ]
  }
>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 25,
        "fontWeight": "bold",
        "marginBottom": 20,
        "textAlign": "center",
      }
    }
  >
    Hello there
  </Text>
  <Text
    style={
      Object {
        "color": "#ffffff",
        "fontSize": 15,
      }
    }
    testID="text"
  >
    We are Team Sikkunt. We have developed GoJim as an integrated booking app for all NUS facilities for our Orbital project. 
    


    Thanks for downloading GoJim. We appreciate any feedback and you may send them over at our emails. 
    


    -Team Sikkunt
    


    Amirul Shyakir  E0725279@u.nus.edu
    

    Muhammad Mursyid  E0725270@u.nus.edu
    


  </Text>
  <Image
    source={1}
    style={
      Object {
        "alignSelf": "center",
        "bottom": 20,
        "height": 50,
        "position": "absolute",
        "resizeMode": "contain",
        "width": 80,
      }
    }
    testID="image"
  />
</View>
`;
